components/record/MetadataField.vue

```diff
@@ -6,14 +6,27 @@
     <div data-qa="metadata field name">
       <strong>{{ name }}</strong>
     </div>
-    <pre data-qa="metadata field value">
+    <LangMap
+      v-if="isLangMap"
+      :value="value"
+      data-qa="metadata field value"
+    />
+    <pre
+      v-else
+      data-qa="metadata field value"
+    >
       <code>{{ value }}</code>
```

Comment:
if we'll render mapped values inside `<span :lang="mapValue.lang">xyz</span>`, then let's render literal values as `<span :lang="und">xyz</span>` too.

Reply:
or perhaps just inside a span, language could set on the record as a whole.

Reply:
Switched to a `div` without `lang` attribute.